### PR TITLE
[BACKLOG-5799] Report Viewer - will not show a special pop-up if ther…

### PR DESCRIPTION
**Pull request by annapopova1**
…e is a fatal error

Fixed 2 problems:
- incorrect code for setting style to elements. "dojo/dom-style" works with single domNode but "dojo/query" returns nodeList and we should use forEach to set width.
- progress indicator from blockUi is located over the dijit dialog (pentaho.common.Dialog). he reason is blockUi uses 1000 and 1011 z-index by default but dijit dialog uses 950 z-index by default. To fix it I set default z-index for dijit dialog.

@pentaho/orlandocalisbon please review

### CODE REVIEW

File: package-res/resources/web/dojo/pentaho/common/Dialog.js [hunk 1]
```diff
@@ -32,6 +32,10 @@ define(["dojo/_base/declare", "dijit/_WidgetBase", "dijit/_TemplatedMixin", "dij
             closeIcon: undefined,
             _onCancelCallback: undefined,
 
+            constructor: function() {
+              Dialog._DialogLevelManager._beginZIndex = 1051;
```
Comment (diogofscmariano) [inline package-res/resources/web/dojo/pentaho/common/Dialog.js]: Cant we set this via css just for RV/PIR?

Comment (annapopova1) [inline package-res/resources/web/dojo/pentaho/common/Dialog.js]: Looks like we can't.
For dijit/Dialog z-index property is hard-coded in js code (950 level). It overrides all css z-index properties.

**Comment by krivera-pentaho** [comment]
This looks good to me, since you answered diogo's question.

**Comment by diogofscmariano** [comment]
I'm really afraid with this question. Anna, can you please search where this is being used on the platform in order to know what are the tests we need to do?

**Comment by annapopova1** [comment]
Investigation of usages:
common-ui
	pentaho/common/Dialog is parent for:
		pentaho.common.datasourceselect
		pentaho.common.FilterDialog
		pentaho.common.MessageBox
		pentaho.common.SplashScreen
		pentaho.common.TemplatePicker
		
RV
	pentaho.reportviewer.ReportDialog - do not find where is used
	pentaho.common.MessageBox
	
PIR
	pentaho.pir.PageSetupDialog
	pentaho.common.datasourceselect
	pentaho.common.FilterDialog
	pentaho.common.MessageBox
	pentaho.common.SplashScreen
	pentaho.common.TemplatePicker

**Pull request merged**